packages/color/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "@pixi/color",
+    "version": "7.1.0",
+    "main": "lib/index.js",
+    "module": "lib/index.mjs",
+    "types": "lib/index.d.ts",
+    "exports": {
+      ".": {
+        "import": {
+          "types": "./lib/index.d.ts",
+          "default": "./lib/index.mjs"
+        },
+        "require": {
+          "types": "./lib/index.d.ts",
+          "default": "./lib/index.js"
+        }
+      }
+    },
+    "description": "Color utility classes and functions for PixiJS",
+    "author": "Matt Karl",
+    "homepage": "http://pixijs.com/",
+    "bugs": "https://github.com/pixijs/pixijs/issues",
+    "license": "MIT",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/pixijs/pixijs.git"
+    },
+    "publishConfig": {
+      "access": "public"
+    },
+    "files": [
+      "lib",
+      "*.d.ts"
+    ],
+    "devDependencies": {
+      "colord": "^2.9.3"
+    }
```

Comment:
By the way, why is `colord` (and `css-color-names` in original code) a dev dependency? It should be a normal dependency in my opinion.